understanding/focus-appearance.html

```diff
@@ -445,498 +372,379 @@ <h4>Inline links</h4>
-                     around the whole tablist as well as one around a tab item (conventionally the item
-                     that is currently selected). The focus around the whole is helpful in cueing a keyboard
-                     user that this is a complex component that has its own interaction. The user can then
-                     move focus between the unselected and selected tab items -- each of which in turn
-                     has its own focus indicator -- before activating one, which then makes it selected
-                     as well as focused, and updates the tab panel to match.
+                  <p>利用者はタブリストにナビゲートできる。この実装では、タブリスト全体だけでなく、タブ項目 (慣習的に、現在選択されている項目) の周囲にもフォーカス矩形がある。全体を囲むフォーカスは、キーボードの利用者に、これが独自のインタラクションを持つ複雑なコンポーネントであることを知らせるのに役立つ。利用者は、選択されていないタブ項目と選択されたタブ項目 (それぞれが独自のフォーカスインジケーターがある) の間でフォーカスを移動でき、その後、一つのタブ項目をアクティブにする前に、そのタブ項目を選択してフォーカスし、タブパネルを更新して一致させることができる。
                   </p>
                   
-                  <p>In this scenario, either the group focus indicator or the sub-component indicator
-                     must meet the requirements of this Success Criterion. To avoid being overly prescriptive,
-                     the Success Criterion allows authors to choose which makes the most sense. Generally,
-                     if a sub-component focus is necessary, it should be assessed instead of the group
-                     indicator.
+                  <p>このシナリオでは、グループフォーカスインジケータ又はサブコンポーネントインジケータのいずれかが、この達成基準の要件を満たさなければならない。過度に規範的にならないように、この達成基準では、コンテンツ制作者が最も適切なものを選択できるようにしている。一般に、サブコンポーネントフォーカスが必要な場合、グループインジケータの代わりにサブコンポーネントフォーカスを評価すべきである。
                   </p>
                   
                   
-                  <p>Result: the focus indicator for the tab item meets the requirements of this Success
-                     Criterion. The tablist focus indicator does not need to meet the requirements.
+                  <p>結果: タブ項目のフォーカスインジケータは、この達成基準の要件を満たしている。タブリストのフォーカスインジケータは、要件を満たす必要はない。
                   </p>
                   
                   
-                  <p>A <a href="https://www.w3.org/WAI/ARIA/apg/example-index/slider/slider-color-viewer.html">slider to pick colors</a> provides a working example of a different complex component that predominantly shows
-                     focus for the subcomponent. In this case, the thumb slider sub-component has a focus
-                     indicator of sufficient size and contrast to pass the sufficient area calculation.
-                     There is also a subtle focus around the whole slider component, but it does not need
-                     to be assessed to pass this Success Criterion.
+                  <p><a href="https://www.w3.org/WAI/ARIA/apg/example-index/slider/slider-color-viewer.html">色を選択するスライダー</a>は、主にサブコンポーネントのフォーカスを表示する別の複雑なコンポーネントの実例を提供する。この場合、つまみのスライダーサブコンポーネントには、十分な面積の計算に合格するのに十分なサイズ及びコントラストのフォーカスインジケータがある。スライダーコンポーネント全体の周囲にもかすかなフォーカスがあるが、この達成基準に合格するために評価する必要はない。
```

Comment:
sufficient area calculation の訳は、「十分な領域の計算」でよいと思います。ここを「面積」にしてしまうと、あとに続く「合格するのに十分なサイズ**及びコントラスト**のフォーカスインジケータがある。」とのつながりが悪く見えます。

```suggestion
                  <p><a href="https://www.w3.org/WAI/ARIA/apg/example-index/slider/slider-color-viewer.html">色を選択するスライダー</a>は、主にサブコンポーネントのフォーカスを表示する別の複雑なコンポーネントの実例を提供する。この場合、つまみのスライダーサブコンポーネントには、十分な領域の計算に合格するのに十分なサイズ及びコントラストのフォーカスインジケータがある。スライダーコンポーネント全体の周囲にもかすかなフォーカスがあるが、この達成基準に合格するために評価する必要はない。
```